Userland/Demos/Mandelbrot/Mandelbrot.cpp

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
```

Comment:
I don't recall working on this :^)

Reply:
Well, it contains a bunch of code from the `Fire` demo though. :)

Reply:
:^)